spec/controllers/givers_controller_spec.rb

```diff
@@ -11,10 +11,16 @@
   describe "POST create" do
     context "with valid info" do
       before do
+        transaction = double('transaction')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.